tests/tensorflow2/test_tensorflow2_datatypes.py

```diff
@@ -0,0 +1,16 @@
+# Third Party
+from tensorflow.python.framework.dtypes import _NP_TO_TF
+
+# First Party
+from smdebug.core.tfevent.util import _get_proto_dtype
+
+
+def test_tensorflow2_datatypes():
```

Comment:
apart from this unit test, is there a training script that makes use of these different datatypes, that we can use for testing?

Reply:
The this test tracks all possible TF type to numpy types.

Any update to the mapping will make this test break, effectively helping us track possible tf to numpy type conversation.

Reply:
I guess we should also test, if these new types tensors are saved they are saved without errors. 
You should create tensors and may be try using save_tensor API